openwisp_radius/api/views.py

```diff
@@ -419,15 +453,24 @@ def get(self, request, *args, **kwargs):
 
 
 class RadiusTokenMixin(object):
-    def get_or_create_radius_token(self, user):
-        radius_token, _ = RadiusToken.objects.get_or_create(user=user)
+    def get_or_create_radius_token(self, user, organization, enable_auth=False):
+        try:
+            radius_token, _ = RadiusToken.objects.get_or_create(
+                user=user, organization=organization
+            )
+        except IntegrityError:
+            radius_token = RadiusToken.objects.get(user=user)
+            radius_token.organization = organization
```

Comment:
When user is using `org1` and sends a `user_auth_token` request to org2, the organization is changed, I think we also need to send accounting `Stop` request to `org1`, right? Or is the request send from elsewhere when user stops using the connection (like captive portal, freeradius etc)?

Reply:
we can check if the user has an open session (RadiusAccounting), retrieve it and flag it as terminated, as termination cause we have to choose the best value from this list: https://tools.ietf.org/html/rfc2866#page-19

Reply:
```
NAS Request          NAS ended session for a non-error reason not otherwise listed here.
```

This looks good to me! :smile:

Reply:
yes, we can use this termination cause to close active user sessions

Reply:
Worked on it.